src/Helper/HtmlFormErrors.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Form\Helper;
+
+use Yiisoft\Form\FormModelInterface;
+
+/**
+ * HtmlFormErrors renders a list of errors for the specified model attribute.
+ */
+final class HtmlFormErrors
+{
+    /**
+     * Returns the errors for single attribute.
+     *
+     * @param FormModelInterface $formModel the form object.
+     * @param string $attribute attribute name. Use null to retrieve errors for all attributes.
+     */
+    public static function getError(FormModelInterface $formModel, string $attribute): array
+    {
+        return $formModel->getFormErrors()->getError($attribute);
+    }
+
+    /**
+     * Returns the errors for all attributes.
+     *
+     * @param FormModelInterface $formModel the form object.
+     *
+     * @return array the error messages.
+     */
+    public static function getErrors(FormModelInterface $formModel): array
+    {
+        return $formModel->getFormErrors()->getErrors();
+    }
+
+    /**
+     * Returns the errors for all attributes as a one-dimensional array.
+     *
+     * @param FormModelInterface $formModel the form object.
+     * @param bool $showAllErrors boolean, if set to true every error message for each attribute will be shown otherwise
+     * only the first error message for each attribute will be shown.
+     *
+     * @return array errors for all attributes as a one-dimensional array. Empty array is returned if no error.
+     */
+    public static function getErrorSummary(FormModelInterface $formModel, bool $showAllErrors): array
```

Comment:
I'd introduce another method instead of a boolean flag.

Reply:
done.